### ISSUE TITLE
[v0.9.x] Queue for plugin installation

### ISSUE THREAD

**Issue by thymian** [pimatic/pimatic#919]
I just installed a fresh 0.9.18 on my pi.
I went into the plugin section and scrolled through the whole list. Sadly I cant go though it and click on every plugin I want to install, because I always have to wait until the last one is installed.

Plugin-Installations would be more intuitive if one:
1) present a bigger popup "Plugin is installing, please wait"
or 2) BETTER: create a queue for every plugin where i pressed "Install" and install them one after another.

**Comment by mwittig** [comment]
Agree, this should be improved. If I try to install another plugin, say `pimatic-edimax`, while a plugin installation is in progress I get the following error:
```
error [pimatic]: Error installing plugin pimatic-edimax: undefined
```
